content/secrets-management/cicd.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Using Secrets with CI/CD"
+description: "Learn how we recommend manaing secrets as part of CI/CD pipelines."
+tags:
+- ci/cd
+- codefresh
+- secrets
+---
+
+We advocate using `chamber` with all CI/CD pipelines to access secrets. Chamber provides a universal interface and a single "system of record" for storing and rotating secrets.
+
+Enabling CI/CD systems to access chamber depends on the CI/CD solution. For CodeBuild/CodePipeline, it's possible to leverage IAM Roles to grant access to KMS+SSM, while for other systems which are external to AWS, it's necessary to [provision a chamber IAM user](https://github.com/cloudposse/terraform-aws-iam-chamber-user) with a very limited scope (the link provides an example for how we do this using `terraform`).
+
+# Using Chamber with CodeFresh
+
+If using codefresh, there a few things that need to happen.
```

Comment:
CodeFresh (capitalize) for consistency?

Reply:
there are few things

Reply:
Yea, looks like "Codefresh" is the official spelling. I will update it everywhere.